dali/base/dasubs.cpp

```diff
@@ -605,6 +607,8 @@ class CDaliPublisherClient: public Thread, public CDaliPublisher
     {
         //ICoven &coven=queryCoven();
         //ICommunicator &comm=coven.queryComm();
+        if (stopped)
+            return;
```

Comment:
processMessage called by handleMessage, so shouldn't of got this far if stopped==true.

Equally if could be set in window in between(?) then could be set after this check.